compiler/rustc_codegen_cranelift/src/driver/aot.rs

```diff
@@ -728,26 +728,34 @@ pub(crate) fn run_aot(
 
     let concurrency_limiter = IntoDynSyncSend(ConcurrencyLimiter::new(todo_cgus.len()));
 
-    let modules = tcx.sess.time("codegen mono items", || {
-        let mut modules: Vec<_> = par_map(todo_cgus, |(_, cgu)| {
-            let dep_node = cgu.codegen_dep_node(tcx);
-            tcx.dep_graph
-                .with_task(
-                    dep_node,
-                    tcx,
-                    (global_asm_config.clone(), cgu.name(), concurrency_limiter.acquire(tcx.dcx())),
-                    module_codegen,
-                    Some(rustc_middle::dep_graph::hash_result),
+    let modules: Vec<_> =
+        tcx.sess.time("codegen mono items", || {
+            let modules: Vec<_> = par_map(todo_cgus, |(_, cgu)| {
+                let dep_node = cgu.codegen_dep_node(tcx);
+                IntoDynSyncSend(
+                    tcx.dep_graph
+                        .with_task(
+                            dep_node,
+                            tcx,
+                            (
+                                global_asm_config.clone(),
+                                cgu.name(),
+                                concurrency_limiter.acquire(tcx.dcx()),
+                            ),
+                            module_codegen,
+                            Some(rustc_middle::dep_graph::hash_result),
+                        )
+                        .0,
                 )
```

Comment:
```suggestion
                let (module, _) = tcx.dep_graph
                    .with_task(
                        dep_node,
                        tcx,
                        (global_asm_config.clone(), cgu.name(), concurrency_limiter.acquire(tcx.dcx())),
                        module_codegen,
                        Some(rustc_middle::dep_graph::hash_result),
                    );
                IntoDynSyncSend(module)
```

reduces indentation and avoids rustfmt putting the `tcx.sess.time()` call on a separate indented line.